### PR TITLE
Update seatmap

### PR DESCRIPTION
**Pull request by FrancisBehnen**
Update seat map to the layout of this year

### CODE REVIEW

**Comment by FrancisBehnen** [comment]
I'm not sure what to do with the inline styling. Is this okay or do we want to do this another way? Does someone have a suggestion to do this nicer?

**Comment by TimvdLippe** [comment]
Please put the inline styles as normal CSS in the component

**Comment by martijnjanssen** [comment]
Why don't you just put the styling in the style section with the rest of the css?

**Comment by FrancisBehnen** [comment]
But how? Give it all id's?

**Comment by TimvdLippe** [comment]
You usually use classes for that. You could in theory also use an attribute selector, but just put a class on it

File: src/lancie-seatmap/lancie-seatmap.html [hunk 1]
```diff
@@ -35,6 +35,27 @@
                   0 1px 5px 0 rgba(0, 0, 0, 0.12),
                   0 3px 1px -2px rgba(0, 0, 0, 0.2);
     }
+
+    .seats {
+      height: 60vh;
+    }
+
+    .lounge {
+      height:500px;
```
Comment (martijnjanssen) [inline src/lancie-seatmap/lancie-seatmap.html]: Please place a space after the `:`.

**Comment by MatthijsKok** [comment]
Please add a few pixels of space between the images. Rest looks good

**Comment by MatthijsKok** [comment]
LGTM

**Pull request merged**